### DISCUSSION TITLE
Error after merging chapters

### DISCUSSION THREAD

**Discussion by Lentilbake** [yermak/AudioBookConverter#359] [General]
I'm new to AudioBookConverter and I'm trying to merge 46 MP3 file into one M4B. I'm not bothered about chapters really, I just want the big single file. I drop the sequentially named files into the window, select them and choose combine from the chapter menu. After the first the program the continues 'Merging files' but gives an error at the end.
![Capture](https://user-images.githubusercontent.com/96791409/147594982-54097842-c20a-4f32-894a-d10bdfede25b.PNG)
.
I tried the same thing with a different set of 4 MP3 files and it worked fine.

Any thoughts?

**Comment by yermak** [comment]
If you just need single resulting file - you can ignore playing with chapters.
Just add files and press start, please let me know if it helps.
Alternatively upload files somewhere to cloud and send me a link to %my_nick_name% @ gmail.com

Finding reason will be useful anyway, however, new version is coming (hopefully tomorrow) there are a lot of stability improvements.

**Reply by Lentilbake** [reply]
I tried just adding the files, selecting them and pressing start but the same thing happens, the Error! message appears after merging and no file is written. I also tried joining all the small MP3s together (in another App) into one big file (AudioBookConverter shows this as having one chapter) but the merging still fails with Error! and fails to write the M4B file. 

Something odd about the MP3 maybe?

I tried converting some other MP3s and they failed too, I seem to have just got lucky with the first four I tried.

I notice that I'm being offered a new version when I open the App and I'm being directed to the Steam website. I have version 5.6.2 which I didn't pay for (not sure where I got it from). Happy to pay for it if it's going to work, what is the latest version (showing 4 on team website). What is Steam? do I have to have it to use this software?

**Comment by fabwardo** [comment]
I am also having this issue. I have used the previous version flawlessly, but since 5.6.2 I get Error! after merging, and it doesn't write to the selected folder (have tried internal and external folders if that was the issue). I'm doing nothing different in this version than I was previously, and it worked flawlessly earlier. Only difference is that I'm on Win11 now rather than 10.

EDIT! 
I just wound back to 5.6.1 and it works fine, so it's something in the latest version. Not sure where to get an error report or logs (I'm new to Windows after a decades as a Mac user).

**Comment by yermak** [comment]
Duplicates #353
Fix is coming today

**Reply by Lentilbake** [reply]
Thanks! Can you give a link for the proper download. I'm happy to pay for it but I'm a bit confused as to which download to use.

**Comment by yermak** [comment]
Hi,
you can download this program for free here https://github.com/yermak/AudioBookConverter/releases - this is open source program and couple of other people contributed to it.
...Or you could buy it from steam to get some extra convenience (automatic updates, some confidence that program is safe, etc) https://store.steampowered.com/app/1529240/AudiobookConverter/

Features are same (at least for now and nearest future). 

It may look strange and confusing, but that's the way I decided to distribute this program.

So, you can choose what works better for you.

**Comment by yermak** [comment]
BTW, new version was released this morning, which should solve quite a few issues.
